python/setup.py

```diff
@@ -25,15 +25,7 @@
 configuration = os.getenv('CPP_CONFIGURATION', "Release")
 
 # Define Python packages to be included
-packages = [
-    'Ice',
-    'IceMX',
-    'slice',
-    'slice.Glacier2',
-    'slice.Ice',
-    'slice.IceBox',
-    'slice.IceGrid',
-    'slice.IceStorm']
+packages = ['Ice', 'IceMX', 'slice']
```

Comment:
A single "slice" pseudo-package is enough to install all `.ice` files. Assuming we still want to include all the Slice files with the package.

Reply:
It makes sense to ship Identity.ice and other Slice files compiled in the package.

It makes less sense to ship the Slice files for IceGrid, IceStorm, Glacier2 etc.

Reply:
> It makes less sense to ship the Slice files for IceGrid, IceStorm, Glacier2 etc.

How do you see us providing a Glacier2 example in this case? We could copy the required Glacier2 Slice files to the examples repository.

Reply:
Will clean-up in a follow-up PR